ls1intum/Artemis
# `Development`: Fix usage of SERVER_API_URL

### Checklist
#### General


#### Client

### Motivation and Context


While experimenting locally with the `SERVER_API_URL` environment variable I noticed that some endpoints did no longer load. 

### Description

This was due to the fact that the `SERVER_API_URL` is supposed to end with a `/`, but some services expected it not to. To fix this I removed the unnecessary leading `/` in these services and additionally added a comment to the declaration of `SERVER_API_URL`, so this is clear in the future.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate around Artemis and check if everything works as expected
    - Emojis load without issue
    - Submission Policy
    - Build log

If you actually want to test setting the `SERVER_API_URL` you can do the following (locally only):
1. Set `SERVER_API_URL` in `webpack/environment.js` to `https://artemis.ase.in.tum.de`
2. Start the client
3. Open the angular application in your browser
    - Note: You might need to disable CORS in your browser (I did not get this to work in Firefox)
5. You can now navigate around Artemis production with your local client

### Review Progress





#### Code Review
#### Manual Tests
